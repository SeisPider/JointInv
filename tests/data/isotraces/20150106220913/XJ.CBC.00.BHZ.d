SURF96 R U X -1 80.00000 3.59852 0.18089
SURF96 R U X -1 28.00000 3.09461 0.04682
SURF96 R U X -1 55.00000 3.57183 0.12252
SURF96 R U X -1 94.99999 4.10177 0.27908
SURF96 R U X -1 29.00000 3.39224 0.05827
SURF96 R U X -1 85.00000 3.71271 0.20458
SURF96 R U X -1 36.00000 3.32886 0.06966
SURF96 R U X -1 48.00000 3.55929 0.10618
SURF96 R U X -1 30.00000 3.13034 0.05133
SURF96 R U X -1 65.00000 3.75679 0.16018
SURF96 R U X -1 46.00000 3.57840 0.10285
SURF96 R U X -1 26.00000 3.13777 0.04470
SURF96 R U X -1 50.00000 3.86215 0.13023
SURF96 R U X -1 50.00000 3.62780 0.11490
SURF96 R U X -1 29.00000 3.10046 0.04868
SURF96 R U X -1 27.00000 3.10662 0.04550
SURF96 R U X -1 44.00000 3.59359 0.09922
SURF96 R U X -1 70.00000 3.54622 0.15371
SURF96 R U X -1 40.00000 3.43949 0.08263
SURF96 R U X -1 42.00000 3.53798 0.09180
SURF96 R U X -1 48.00000 3.85454 0.12452
SURF96 R U X -1 75.00000 3.44036 0.15500
SURF96 R U X -1 29.00000 3.31989 0.05581
SURF96 R U X -1 32.00000 3.23224 0.05837
SURF96 R U X -1 25.00000 3.15201 0.04337
SURF96 R U X -1 44.00000 3.23779 0.08054
SURF96 R U X -1 60.00000 3.58133 0.13437
SURF96 R U X -1 34.00000 3.28915 0.06423
SURF96 R U X -1 38.00000 3.37494 0.07558
SURF96 R U X -1 90.00000 3.96594 0.24718
