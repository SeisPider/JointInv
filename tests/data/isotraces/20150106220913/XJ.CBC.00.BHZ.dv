MFT96 R U T -1 95.256    3.5662    1.0000    5727.0     0.5209E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 93.091    3.5703    1.0000    5727.0     0.6060E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 91.022    3.5741    1.0000    5727.0     0.6957E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 89.043    3.5776    1.0000    5727.0     0.7883E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 87.149    3.5808    1.0000    5727.0     0.8823E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 85.333    3.5837    1.0000    5727.0     0.9758E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 83.592    3.5863    1.0000    5727.0     0.1067E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 81.920    3.5887    1.0000    5727.0     0.1153E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 80.314    3.5908    1.0000    5727.0     0.1233E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 78.769    3.5926    1.0000    5727.0     0.1303E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 77.283    3.5941    1.0000    5727.0     0.1363E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 75.852    3.5954    1.0000    5727.0     0.1411E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 74.473    3.5963    1.0000    5727.0     0.1444E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 73.143    3.5971    1.0000    5727.0     0.1462E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 71.860    3.5975    1.0000    5727.0     0.1464E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 70.621    3.5977    1.0000    5727.0     0.1449E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 69.424    3.5976    1.0000    5727.0     0.1418E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 68.267    3.5973    1.0000    5727.0     0.1372E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 67.148    3.5968    1.0000    5727.0     0.1310E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 66.065    3.5960    1.0000    5727.0     0.1235E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 65.016    3.5949    1.0000    5727.0     0.1149E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 64.000    3.5936    1.0000    5727.0     0.1053E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 63.015    3.5921    1.0000    5727.0     0.9494E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 62.061    3.5904    1.0000    5727.0     0.8417E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 61.134    3.5884    1.0000    5727.0     0.7322E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 60.235    3.5862    1.0000    5727.0     0.6244E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 59.362    3.5838    1.0000    5727.0     0.5220E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 58.514    3.5812    1.0000    5727.0     0.4297E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 57.690    3.5784    1.0000    5727.0     0.3542E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 56.889    3.5754    1.0000    5727.0     0.3039E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 56.110    3.5721    1.0000    5727.0     0.2860E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 55.351    3.5687    1.0000    5727.0     0.2993E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 54.613    3.5651    1.0000    5727.0     0.3335E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 53.895    3.5614    1.0000    5727.0     0.3773E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 53.195    3.5574    1.0000    5727.0     0.4231E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 52.513    3.5533    1.0000    5727.0     0.4665E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 51.848    3.5490    1.0000    5727.0     0.5053E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 51.200    3.5445    1.0000    5727.0     0.5384E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 50.568    3.5399    1.0000    5727.0     0.5659E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 49.951    3.5352    1.0000    5727.0     0.5885E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 49.349    3.5302    1.0000    5727.0     0.6080E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 48.762    3.5252    1.0000    5727.0     0.6272E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 48.188    3.5200    1.0000    5727.0     0.6507E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 47.628    3.5146    1.0000    5727.0     0.6842E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 47.080    3.5092    1.0000    5727.0     0.7347E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 46.545    3.5036    1.0000    5727.0     0.8087E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 46.022    3.4979    1.0000    5727.0     0.9116E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 45.511    3.4921    1.0000    5727.0     0.1046E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 45.011    3.4861    1.0000    5727.0     0.1214E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 44.522    3.4801    1.0000    5727.0     0.1414E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 44.043    3.4739    1.0000    5727.0     0.1644E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 43.574    3.4677    1.0000    5727.0     0.1902E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 43.116    3.4614    1.0000    5727.0     0.2184E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 42.667    3.4549    1.0000    5727.0     0.2489E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 42.227    3.4484    1.0000    5727.0     0.2811E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 41.796    3.4419    1.0000    5727.0     0.3147E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 41.374    3.4352    1.0000    5727.0     0.3493E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.960    3.4285    1.0000    5727.0     0.3843E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.554    3.4217    1.0000    5727.0     0.4193E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.157    3.4148    1.0000    5727.0     0.4539E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 39.767    3.4079    1.0000    5727.0     0.4876E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 39.385    3.4010    1.0000    5727.0     0.5202E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 39.010    3.3939    1.0000    5727.0     0.5512E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 38.642    3.3869    1.0000    5727.0     0.5806E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 38.280    3.3798    1.0000    5727.0     0.6081E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 37.926    3.3727    1.0000    5727.0     0.6337E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 37.578    3.3655    1.0000    5727.0     0.6575E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 37.236    3.3583    1.0000    5727.0     0.6797E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.901    3.3511    1.0000    5727.0     0.7005E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.571    3.3438    1.0000    5727.0     0.7203E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.248    3.3365    1.0000    5727.0     0.7392E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.930    3.3293    1.0000    5727.0     0.7577E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.617    3.3220    1.0000    5727.0     0.7762E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.310    3.3147    1.0000    5727.0     0.7948E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 35.009    3.3074    1.0000    5727.0     0.8139E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.712    3.3001    1.0000    5727.0     0.8336E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.420    3.2928    1.0000    5727.0     0.8540E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.133    3.2855    1.0000    5727.0     0.8751E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.851    3.2782    1.0000    5727.0     0.8967E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.574    3.2709    1.0000    5727.0     0.9188E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.301    3.2636    1.0000    5727.0     0.9410E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 33.032    3.2564    1.0000    5727.0     0.9629E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.768    3.2491    1.0000    5727.0     0.9840E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.508    3.2419    1.0000    5727.0     0.1004E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.252    3.2348    1.0000    5727.0     0.1021E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.000    3.2276    1.0000    5727.0     0.1037E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.752    3.2205    1.0000    5727.0     0.1048E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.508    3.2134    1.0000    5727.0     0.1056E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.267    3.2064    1.0000    5727.0     0.1060E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 31.030    3.1994    1.0000    5727.0     0.1060E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.797    3.1924    1.0000    5727.0     0.1055E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.567    3.1855    1.0000    5727.0     0.1046E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.341    3.1787    1.0000    5727.0     0.1034E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.118    3.1718    1.0000    5727.0     0.1020E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.898    3.1651    1.0000    5727.0     0.1005E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.681    3.1584    1.0000    5727.0     0.9916E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.468    3.1517    1.0000    5727.0     0.9804E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.257    3.1451    1.0000    5727.0     0.9733E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.050    3.1386    1.0000    5727.0     0.9713E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.845    3.1321    1.0000    5727.0     0.9750E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.643    3.1257    1.0000    5727.0     0.9841E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.444    3.1194    1.0000    5727.0     0.9977E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.248    3.1131    1.0000    5727.0     0.1014E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.055    3.1069    1.0000    5727.0     0.1032E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.864    3.1008    1.0000    5727.0     0.1049E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.676    3.0948    1.0000    5727.0     0.1063E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.490    3.0888    1.0000    5727.0     0.1073E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.307    3.0829    1.0000    5727.0     0.1078E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.126    3.0771    1.0000    5727.0     0.1075E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.947    3.0714    1.0000    5727.0     0.1066E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.771    3.0658    1.0000    5727.0     0.1049E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.597    3.0602    1.0000    5727.0     0.1026E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.426    3.0548    1.0000    5727.0     0.9957E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.256    3.0494    1.0000    5727.0     0.9599E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.089    3.0441    1.0000    5727.0     0.9193E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 25.924    3.0390    1.0000    5727.0     0.8748E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 25.761    3.0339    1.0000    5727.0     0.8274E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 25.600    3.0289    1.0000    5727.0     0.7779E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 25.441    3.0240    1.0000    5727.0     0.7271E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 25.284    3.0192    1.0000    5727.0     0.6756E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 25.129    3.0146    1.0000    5727.0     0.6240E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 95.000    3.5667    1.0000    5727.0     0.5309E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 90.000    3.5759    1.0000    5727.0     0.7435E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 85.000    3.5842    1.0000    5727.0     0.9932E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 80.000    3.5911    1.0000    5727.0     0.1247E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 75.000    3.5960    1.0000    5727.0     0.1431E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 70.000    3.5977    1.0000    5727.0     0.1433E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 65.000    3.5949    1.0000    5727.0     0.1147E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 60.000    3.5856    1.0000    5727.0     0.5968E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 55.000    3.5670    1.0000    5727.0     0.3155E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 50.000    3.5355    1.0000    5727.0     0.5867E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 48.000    3.5182    1.0000    5727.0     0.6619E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 46.000    3.4976    1.0000    5727.0     0.9175E-06   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 44.000    3.4734    1.0000    5727.0     0.1667E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 42.000    3.4450    1.0000    5727.0     0.2988E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 40.000    3.4121    1.0000    5727.0     0.4675E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 38.000    3.3741    1.0000    5727.0     0.6283E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 36.000    3.3309    1.0000    5727.0     0.7537E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 34.000    3.2820    1.0000    5727.0     0.8853E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 32.000    3.2276    1.0000    5727.0     0.1037E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 30.000    3.1682    1.0000    5727.0     0.1012E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 29.000    3.1370    1.0000    5727.0     0.9722E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 28.000    3.1052    1.0000    5727.0     0.1037E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 27.000    3.0731    1.0000    5727.0     0.1069E-04   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
MFT96 R U T -1 26.000    3.0414    1.0000    5727.0     0.8953E-05   55.204899  163.244598   43.521099   81.439201 0  1 COMMENT
