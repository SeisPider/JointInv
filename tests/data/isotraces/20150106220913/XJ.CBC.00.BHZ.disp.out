SURF96 R U T -1 95.000    3.5667    1.0000    
SURF96 R C T -1 95.000    6.4004    1.0000    
SURF96 R U T -1 90.000    3.5759    1.0000    
SURF96 R C T -1 90.000    6.1444    1.0000    
SURF96 R U T -1 85.000    3.5842    1.0000    
SURF96 R C T -1 85.000    5.9091    1.0000    
SURF96 R U T -1 80.000    3.5911    1.0000    
SURF96 R C T -1 80.000    5.6924    1.0000    
SURF96 R U T -1 75.000    3.5960    1.0000    
SURF96 R C T -1 75.000    5.4920    1.0000    
SURF96 R U T -1 70.000    3.5977    1.0000    
SURF96 R C T -1 70.000    5.3057    1.0000    
SURF96 R U T -1 65.000    3.5949    1.0000    
SURF96 R C T -1 65.000    5.1315    1.0000    
SURF96 R U T -1 60.000    3.5856    1.0000    
SURF96 R C T -1 60.000    4.9675    1.0000    
SURF96 R U T -1 55.000    3.5670    1.0000    
SURF96 R C T -1 55.000    4.8116    1.0000    
SURF96 R U T -1 50.000    3.5355    1.0000    
SURF96 R C T -1 50.000    4.6614    1.0000    
SURF96 R U T -1 48.000    3.5182    1.0000    
SURF96 R C T -1 48.000    4.6021    1.0000    
SURF96 R U T -1 46.000    3.4976    1.0000    
SURF96 R C T -1 46.000    4.5431    1.0000    
SURF96 R U T -1 44.000    3.4734    1.0000    
SURF96 R C T -1 44.000    4.4840    1.0000    
SURF96 R U T -1 42.000    3.4450    1.0000    
SURF96 R C T -1 42.000    4.4244    1.0000    
SURF96 R U T -1 40.000    3.4121    1.0000    
SURF96 R C T -1 40.000    4.3640    1.0000    
SURF96 R U T -1 38.000    3.3741    1.0000    
SURF96 R C T -1 38.000    4.3025    1.0000    
SURF96 R U T -1 36.000    3.3309    1.0000    
SURF96 R C T -1 36.000    4.2393    1.0000    
SURF96 R U T -1 34.000    3.2820    1.0000    
SURF96 R C T -1 34.000    4.1738    1.0000    
SURF96 R U T -1 32.000    3.2276    1.0000    
SURF96 R C T -1 32.000    4.1056    1.0000    
SURF96 R U T -1 30.000    3.1682    1.0000    
SURF96 R C T -1 30.000    4.0340    1.0000    
SURF96 R U T -1 29.000    3.1370    1.0000    
SURF96 R C T -1 29.000    3.9968    1.0000    
SURF96 R U T -1 28.000    3.1052    1.0000    
SURF96 R C T -1 28.000    3.9585    1.0000    
SURF96 R U T -1 27.000    3.0731    1.0000    
SURF96 R C T -1 27.000    3.9191    1.0000    
SURF96 R U T -1 26.000    3.0414    1.0000    
SURF96 R C T -1 26.000    3.8786    1.0000    
